src/libsyntax/ast.rs

```diff
@@ -1584,8 +1569,7 @@ pub struct EnumDef {
 pub struct Variant_ {
     pub name: Ident,
     pub attrs: Vec<Attribute>,
-    pub kind: VariantKind,
-    pub id: NodeId,
+    pub def: P<StructDef>,
```

Comment:
def is a bad name, since we use `def` a lot in the compiler for various other things. `data` maybe?